ietf/templates/meeting/interim_meeting_cancellation_notice.txt

```diff
@@ -3,6 +3,6 @@ The {{ group.name }} ({{ group.acronym }}) {% if not meeting.city %}virtual {% e
 interim meeting for {{ meeting.date|date:"Y-m-d" }} from {{ start_time|time:"H:i" }} to {{ end_time|time:"H:i" }} {{ meeting.time_zone }}
 has been cancelled.
 
-{{ meeting.session_set.0.agenda_note }}
+{{ meeting.session_set.first.agenda_note }}
```

Comment:
Note: I'm deeply uneasy about this change, since I have to assume it worked before, but in my test environment, generated email had the line: ```** No value found for &#x27;meeting.session_set.0.agenda_note&#x27; **'```

Reply:
Why assume it worked before? There's no test that looks to see if the note actually made it into the email i think.
Please add one.

I verified that the cancellation that led to this issue probably had a agenda_note added, but the email that went out did not contain it.